tomviz/pybind11/eigenConversion.h

```diff
@@ -0,0 +1,770 @@
+/*
+    Transparent conversion for dense and sparse Eigen matrices
+
+    This function is adapted from the pybind11 code by Wenzel Jacob.
+    pybind11 - https://github.com/pybind/pybind11
+
+    Copyright (c) 2016 Wenzel Jakob <wenzel.jakob@epfl.ch>
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+
+#pragma once
```

Comment:
We don't use `#pragma once` in our code, header guards.

Reply:
@cryos ah, yes let me double check this line but I believe it's unnecessary and will still function properly if we completely remove it